cli/cmd/common/common.go

```diff
@@ -27,7 +27,7 @@ const (
 
 var (
 	TimescaleDBBackUpKeyForValuesYaml = []string{"timescaledb-single", "backup", "enabled"}
-	PrometheusLabels                  = map[string]string{"app": "prometheus", "prometheus": "tobs-kube-prometheus-prometheus"}
+	PrometheusLabels                  = map[string]string{"app.kubernetes.io/managed-by": "prometheus-operator", "prometheus": "tobs-kube-prometheus-prometheus"}
```

Comment:
The new version of the Prometheus operator changed the labels of the Prometheus pod changing them accordingly.

Reply:
This makes it incompatible with previous operator versions. Would it affect upgrades?

Side note: the intended replacement for `app` label is `app.kubernetes.io/name: "prometheus"` and not `app.kubernetes.io/managed-by`. Ideally, both should be used to select prometheus pods managed by the operator.

Reply:
It supports backward compatibility I switch from changing the new label key to moving to an undisturbed label set which matches both old and new versions. :)